docs/docs/latest/tutorials/getting-started.md

```diff
@@ -63,9 +63,9 @@ Windows. For other options, such as Docker, read the [detailed instructions] (/d
 
 </summary>
 
-1. Download the `edgectl` file
+1. Download the `edgectl.exe` file
    [here](https://metriton.datawire.io/downloads/windows/edgectl.exe).
-2. Run the installer with `edgectl.exe install`
```

Comment:
This seems reasonable but I forget how Windows works: can you run a program without the exe on the end?

Reply:
this is no longer code in this PR now that I've branched off of release/v1.3.2

Reply:
@bnfb you can indeed run a program on Windows without specifying the .exe extension.